db/seeds.rb

```diff
@@ -3,7 +3,13 @@
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 return unless Rails.env.development?
 
-FactoryGirl.create_list(:user, 10, :with_posts)
+@users = []
```

Comment:
why do you need instance variable here?

Reply:
because when I will run seeds next time, they will create factories to the existed posts and comments. I tried to to that, and get about 300 thousand comments

Reply:
seeds must be independent to existed records
![screen shot 2017-11-16 at 6 12 37 pm](https://user-images.githubusercontent.com/23212735/32901950-cfe8a698-caf9-11e7-9ed8-140f9c0ce75b.png)